### PR TITLE
rgb() and rgba() support added

### LINKED ISSUE #1775: Using rgba in CSS
This 8 digit hex color works in the NativeScript framework:
```
.drawer-button {
    background-color: #003751E4;
}
.drawer-button:highlighted {
    background-color: #FF3751E4;
}
```

However we should support something by CSS specs like:
```
.drawer-button {
    background-color: rgba(55, 81, 228, 0);
}
.drawer-button:highlighted {
    background-color: rgba(55, 81, 228, 1);
}
```

However the second throws on button tap:
```
JS ERROR Error: Invalid value rgba(55, 82, 228, 1) for property backgroundColor
```

### PR DESCRIPTION
**Pull request by enchev**
Fixed #1775

### CODE REVIEW

**Comment by dtopuzov** [comment]
:+1:

File: apps/tests/color-tests.ts [hunk 1]
```diff
@@ -73,6 +73,40 @@ export var test_ArgbInt_Color = function () {
...
 
+export var test_rgb_Color_CSS = function () {
+    // <snippet module="color" title="color">
+    // ### Creating a Color from four RGB values
+    // ``` JavaScript
+    //// Creates the color with 255 red, 100 green, 100 blue
+    var color = new Color("rgb(255, 100, 100)");
+    // ```
+    // </snippet>
+    TKUnit.assert(color.a === 255, "Color.a not properly parsed");
```
Comment (hshristov) [inline apps/tests/color-tests.ts]: AssertEqual - returns better error message

**Comment by PanayotCankov** [comment]
:+1:

**Comment by enchev** [comment]
run ci

**Comment by enchev** [comment]
run ci

**Comment by ErjanGavalji** [comment]
:+1:

**Pull request merged**